src/style.css

```
@@ -1,3 +1,77 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+.logo-image {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.nav__link {
+  flex-direction: row;
+  text-decoration: none;
+  color: black;
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  margin-right: 20px;
+  height: 60px;
+  align-items: center;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  vertical-align: baseline;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  display: block;
+  content: '';
+  position: absolute;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+  width: 100%;
+  bottom: 0;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+a:active {
+  color: #00acdc;
+}
```

use class selector instead of tag one
```suggestion
.nav__link:hover,
.nav__link:active  {
  color: #00acdc;
}
```